address/address_v1.go

```diff
@@ -17,15 +17,12 @@ import (
 
 // V1 is a singleton and defines V1 address metadata
 var V1 = v1{
-	AddressLength: 21,
-	Version:       1,
+	AddressLength: 20,
```

Comment:
Do we need AddressLength then?

Reply:
Are we deprecating V1?

Reply:
We are not deprecating V1, and AddressLength is the meta data which identifies V1, so I think we can keep it for now.